scala-package/examples/src/test/scala/org/apache/mxnetexamples/imclassification/IMClassificationExampleSuite.scala

```diff
@@ -31,27 +29,35 @@ import scala.sys.process.Process
 /**
   * Integration test for MNIST example.
   */
-class MNISTExampleSuite extends FunSuite with BeforeAndAfterAll {
-  private val logger = LoggerFactory.getLogger(classOf[MNISTExampleSuite])
+class IMClassificationExampleSuite extends FunSuite with BeforeAndAfterAll {
+  private val logger = LoggerFactory.getLogger(classOf[IMClassificationExampleSuite])
 
   test("Example CI: Test MNIST Training") {
 
-      logger.info("Downloading mnist model")
-      val baseUrl = "https://s3.us-east-2.amazonaws.com/mxnet-scala/scala-example-ci"
-      val tempDirPath = System.getProperty("java.io.tmpdir")
-      val modelDirPath = tempDirPath + File.separator + "mnist/"
-      logger.info("tempDirPath: %s".format(tempDirPath))
-      Util.downloadUrl(baseUrl + "/mnist/mnist.zip",
-        tempDirPath + "/mnist/mnist.zip")
-      // TODO: Need to confirm with Windows
-      Process("unzip " + tempDirPath + "/mnist/mnist.zip -d "
-        + tempDirPath + "/mnist/") !
+    logger.info("Downloading mnist model")
+    val baseUrl = "https://s3.us-east-2.amazonaws.com/mxnet-scala/scala-example-ci"
+    val tempDirPath = System.getProperty("java.io.tmpdir")
+    val modelDirPath = tempDirPath + File.separator + "mnist/"
+    logger.info("tempDirPath: %s".format(tempDirPath))
+    Util.downloadUrl(baseUrl + "/mnist/mnist.zip",
+      tempDirPath + "/mnist/mnist.zip")
+    // TODO: Need to confirm with Windows
+    Process("unzip " + tempDirPath + "/mnist/mnist.zip -d "
+      + tempDirPath + "/mnist/") !
 
-      var context = Context.cpu()
+    var context = Context.cpu()
 
-      val output = TrainMnist.test(modelDirPath)
-      Process("rm -rf " + modelDirPath) !
+    val output = TrainModel.test("mlp", modelDirPath)
+    Process("rm -rf " + modelDirPath) !
 
-      assert(output >= 0.95f)
+    assert(output >= 0.95f)
   }
+
+  for(model <- List("mlp", "lenet", "resnet")) {
```

Comment:
not sure whether this can result timeout in travis CI?

Reply:
Travis currently did not run Scala test. Some Python test break that it looks like.

Reply:
This is just a basic test to determine that each model can run for a single batch